### PR TITLE
upd(pipewire): `0.3.44` -> `0.3.47`

### PR DESCRIPTION
**Pull request by Henryws**

### CODE REVIEW

**Comment by Henryws** [comment]
https://github.com/pacstall/pacstall-programs/runs/5261730506?check_suite_focus=true#step:9:1587

This issue can be fixed by removing apt's meson and installing it from pip. Although I remember this working fine in a VM a while back

File: packages/pipewire/pipewire.pacscript [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
 url="https://gitlab.freedesktop.org/"${name}/"${name}"/-/archive/"${version}"/"${name}-${version}.zip"
 maintainer="Henryws <hwengerstickel@pm.me>"
 description="PipeWire is a server and user space API to deal with multimedia pipelines."
-hash="3ddb9a246fa7c048284851081e08d1b106df491e3b1fcd2bdc9d76460c245f04"
+hash="d1bfca55475d7a215d7425afe0f7eda355a0768f91434496bdd7592a6f205932"
 prepare() {
+    if dpkg -l meson &>/dev/null; then
+        ask "Your meson version is likely outdated, however pip contains a newer version that will compile pipewire correctly. Do you want to remove apt meson and install from pip" Y
+        if [[ $answer -eq 1 ]]; then
+            sudo apt remove meson
```
Comment (wizard-28) [inline packages/pipewire/pipewire.pacscript:12]: why not `purge`?

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Comment by wizard-28** [comment]
WTF happened with the CI.

**Review by wizard-28 — APPROVED** [review]

**Comment by wizard-28** [comment]
I guess since you've requested a review from @D-Brox I'll just enable auto squash for now.

**Pull request merged**